### DISCUSSION TITLE
Help with setting up Playstation on EmulatorJS

### DISCUSSION THREAD

**Discussion by Infrayyy** [EmulatorJS/EmulatorJS#480] [Q&A]
I have used the example code, as well as a working file for my game, but it keeps saying "Firmware is missing: scph5501.bin" i downloaded this and set it as the bios url but this keeps happening. is there a way to fix my problem or should i just not use playstation emulator? thanks!

**Comment by EricWilliamYT** [comment]
what is the bios url
